Makefile

```diff
@@ -314,6 +323,21 @@ yapf:
 	yapf -i t/*.py
 	yapf -i *.py
 
+show-build-env:
```

Comment:
Probably `show-build-env` might be unnecessary to have in the Makefile. I think version information can be get differently depending on the environment and OS, therefore might be good to avoid having environment specific commands in the Makefile.

Reply:
> Probably show-build-env might be unnecessary to have in the Makefile. 

I added `show-build-env` to provide a no-op option in the CI/CD matrix for cases where we want to skip full regression testing (regresscheck isolationcheck testgrescheck) 
for specific Docker image variants. 
This was intended to prevent parameter-related errors when testing is disabled.

However, I agree this addition is unnecessary and can be removed.

Reply:
I think it isn't necessary to have this in the Makefile and can be removed.

Reply:
Ok ; I will remove.